content/en/continuous_delivery/deployments/ciproviders.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Monitor CI Providers Deployments
+kind: documentation
+description: Learn how to monitor deployments from CI Providers in Datadog CD Visibility.
+is_beta: true
+further_reading:
+- link: "/continuous_delivery/deployments"
+  tag: "Documentation"
+  text: "Learn about Deployment Visibility"
+- link: "/continuous_delivery/explorer"
+  tag: "Documentation"
+  text: "Learn how to query and visualize deployment executions"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CD Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" header="false" >}}
+CD Visibility for CI providers deployments is in private beta. To request access, complete the form.
+{{< /callout >}}
+
+## Overview
+
+Deployments can also be performed within continuous integration pipelines. Typically, these pipelines have a deployment step that is executed after the source code has been tested and the image has been built.
+If you are executing deployments through a CI provider, you can monitor your deployments with CD Visibility if the following requirements are met:
+1. You are using [Pipeline Visibility][1] to monitor your CI pipelines.
+2. Your CI provider supports the "Custom tags and metrics at runtime" feature. You can check this in the [supported features table][2].
+3. You are executing deployments in a CI job (or [related level][9] in your CI provider).
+
+## Setup
+
+<div class="alert alert-info">
+The setup requires the <a href="https://www.npmjs.com/package/@datadog/datadog-ci"> <code>datadog-ci</code> CLI </a> with version `2.26.0` or later.
+</div>
+
+To set up CD Visibility, use the `datadog-ci deployment mark` command inside the CI job that is performing the deployment. Two environment variables are required:
+1. `DD_API_KEY`: Point to your [Datadog API key][4].
+2. `DD_BETA_COMMANDS_ENABLED`: Set to 1.
+
+Optionally, you can point the `DD_SITE` environment variable to a specific [Datadog site][3] (default value is `datadoghq.com`).
+
+You can enrich the deployment event that is generated by the `datadog-ci deployment mark` command using the following parameters:
+- `--env`: the environment to which this deployment is performed. For example, `prod`.
+- `--revision`: the revision/version that is being deployed. For example, `1.0.0` or `v123-456`.
+- `--is-rollback` specifies that the deployment is a rollback.
+- `--tags` is an array of key value pairs in the format `key:value`. These tags are added to the deployment event shown in Datadog.
+
+Use `--no-fail` (default: `false`) to prevent the deployment command from failing if there are issues submitting the data.
+
+Once you have added the command to a CI job, the [**Deployments**][5] and [**Deployment Executions**][6] pages populate with data after pipelines are executed.
```

Comment:
```suggestion
Once you have added the command to a CI job, the [**Deployments**][5] and [**Deployment Executions** pages][6] populate with data after pipelines are executed.
```

Reply:
I think "pages" refer to both Deployments and Deployment Executions so I would leave it outside of the link perhaps? Also to be consistent with the current [ArgoCD docs](https://docs.datadoghq.com/continuous_delivery/deployments/argocd#visualize-deployments-in-datadog)

Reply:
Are you ok with referring to the Deployment Executions page as the CD Visibility Explorer?